packages/wp-story-editor/src/api/media.js

```diff
@@ -180,17 +184,24 @@ export function uploadMedia(config, file, additionalData) {
     web_stories_is_muted: isMuted,
     post: templateId || storyId || mediaId,
     original_id: originalId,
-    web_stories_trim_data: trimData,
-    web_stories_base_color: baseColor,
-    web_stories_blurhash: blurHash,
+    meta: {
+      web_stories_base_color: baseColor,
+      web_stories_blurhash: blurHash,
+      web_stories_cropped_origin_id: originalCropId,
+      web_stories_optimized_id: optimizedId,
+      web_stories_muted_id: mutedId,
```

Comment:
I think this is incorrect here, `web_stories_muted_id` is a registered REST field, we don't populate it here. See line 184 above.

Reply:
`web_stories_is_muted` and `web_stories_muted_id` are different fields.